components/infobox/extensions/commons/infobox_extensions_prize_pool.lua

```diff
@@ -0,0 +1,161 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Infobox/Extensions/PrizePool
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Currency = require('Module:Currency')
+local Logic = require('Module:Logic')
+local Namespace = require('Module:Namespace')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+
+local PrizePoolCurrency = {}
+
+local NOW = os.date('!%F')
+local USD = 'USD'
+local LANG = mw.language.new('en')
+local CATEGRORY = '[[Category:Tournaments with invalid prize pool]]'
+
+function PrizePoolCurrency.display(args)
+	local currency = string.upper(args.currency or USD)
+	local date = PrizePoolCurrency._cleanDate(args.date or NOW)
+	local text = args.text or 'Currency exchange rate taken from exchangerate.host'
+	local prizepool = PrizePoolCurrency._cleanValue(args.prizepool)
+	local prizepoolUsd = PrizePoolCurrency._cleanValue(args.prizepoolusd)
+	local currencyRate = tonumber(args.rate)
+	local setVariables = Logic.emptyOr(args.setvariables, true)
+
+	if not prizepool and not prizepoolUsd then
+		if Namespace.isMain() then
+			return (args.prizepool or args.prizepoolUsd or '')
+				.. '[[Category:Tournaments with invalid prize pool]]'
+		else
+			return (args.prizepool or args.prizepoolUsd or '')
+		end
+	end
+
+	if not date then
+		date = NOW
+		mw.log('Infobox league: Invalid currency exchange date -> default date (' .. date .. ')')
+	elseif date > NOW then
+		date = NOW
+	end
+
+	if currency == USD and String.isEmpty(prizepoolUsd) then
+		return PrizePoolCurrency._errorMessage('Need valid currency')
+	elseif currency == USD then
+		prizepoolUsd = LANG:formatNum(math.floor(prizepoolUsd))
+	else
+		local errorMessage
+		errorMessage, prizepool, prizepoolUsd, currencyRate = PrizePoolCurrency._exchange{
```

Comment:
Move `errorMessage` to the last return value

Reply:
done